Kconfig.dependencies

```diff
@@ -38,6 +38,10 @@ config SIDEWALK_BLE
 	help
 		Sidewalk Bluetooth Low Energy (BLE) module
 
+config SIDEWALK_BLE_NAME
+	string "BLE name adverticed for Sidewalk"
+	default "SID_APP"
```

Comment:
I would recommend to left the name for Sidewalk advertising as it was, and rename dfu advertising